src/screens/Settings/Index/index.tsx

```diff
@@ -0,0 +1,287 @@
+import React, { FC, useCallback, useMemo, useEffect } from 'react';
+import {
+  View,
+  Alert,
+  StyleSheet,
+  SafeAreaView,
+  ScrollView,
+} from 'react-native';
+// components
+import Card from 'src/components/Card';
+import Avatar from 'src/components/Avatar';
+import ListItem from 'src/components/ListItem';
+import Typography from 'src/components/Typography';
+// navigation
+import { StackScreenProps } from '@react-navigation/stack';
+import { Routes, RootNavigationStackParamsList } from 'src/navigation';
+// localization
+import { useTranslation } from 'react-i18next';
+// api
+import api from 'src/services/api';
+// store
+import { useDispatch, useSelector } from 'react-redux';
+import { clearAuth, updateAuthUser } from 'src/store/modules/auth';
+// selectors
+import { getAuthUserSelector } from 'src/store/modules/auth/selectors';
+// utils
+import { useAgreementsModals } from 'src/hooks/useAgreementsModals';
+import lodashCapitalize from 'lodash/capitalize';
+// styling
+import { useThemeSchema } from 'src/hooks/useThemeShema';
+
+type SettingsIndexProps = StackScreenProps<
+  RootNavigationStackParamsList,
+  Routes.SignIn
+>;
+
+const SettingsIndex: FC<SettingsIndexProps> = ({ navigation }) => {
+  const { t } = useTranslation();
+  const { colors } = useThemeSchema();
+  const {
+    showPrivacyPolicyModal,
+    showTermsAndConditionsModal,
+  } = useAgreementsModals();
+
+  const dispatch = useDispatch();
+  const authProfile = useSelector(getAuthUserSelector);
+
+  const authProviderName = useMemo(() => authProfile?.provider, [
```

Comment:
I think in this case `useMemo` redundant

Reply:
```ts
const authProviderName = authProfile?.provider;
```

Reply:
U r right :)